packages/buffetjs-core/src/components/Table/index.js

```diff
@@ -67,33 +73,31 @@ function Table({
             </TableRowEmpty>
           )}
           {!shouldDisplayEmptyRow &&
-            rows.map((row, index) => {
-              const key = row.id || `key${index}`;
+          rows.map((row, index) => {
```

Comment:
@soupette I thought about memoizing this as well... because this is `main loop` which is called on every component rerender.

But it should get almost everything as dependency (rows, headers, onSelect, withBulkAction etc). It may be unnecessary checks, so I leave it as is. It may be memoized in the future if there will be some optimization problems with table.

Reply:
Well I will take a look into this one as I really want to improve the api.
I haven't checked the code in a while but I believe that the `withBulkAction` prop and some others are kind of really static i see them as a configuration of the component. So if we use a `useRef` maybe we can remove them from the dependency array.

What do you think?

Reply:
if it not changes during whole components lifecycle but it's set by prop, it's probably the best way to leave it that way. Many components don't change at all during whole lifecycle, so i will not bother with it.

Moving it to ref may be consider as overkill I think.